build.gradle

```diff
@@ -99,7 +99,7 @@ subprojects {
 
     testCompile 'junit:junit:4.12'
     testCompile 'org.slf4j:slf4j-simple:1.7.5'
-    testCompile 'org.mockito:mockito-all:1.10.19'
+    testCompile 'org.mockito:mockito-core:1.10.19'
```

Comment:
`mockito-all` contains mockito classes as well as some dependencies, which might conflict with `junit`.
For example, we cannot use `ExpectedException`.